controller/schema.mustache

```diff
@@ -1,5 +1,7 @@
 {
   "type": "object",
+  "name": "{{name}}",
```

Comment:
Originally this is something I thought i'd need to test relationships (to perform a lookup from the singular to the plural), but I've actually just finished the feature and didn't need it afterall. Though Dave's suggestion to add this was not related to my need, I think he thought it would just be a good idea to store it.